Description: Explicitly use the 1.8 HDF5 API
 Make sure we get the 1.8 HDF5 API by using the right macro
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=13104
Last-Update: 2022-01-03

--- a/configure.ac
+++ b/configure.ac
@@ -221,7 +221,7 @@
 # --with-default-api-version=v16 which creates problems for slurm
 # because slurm uses the 1.8 API. By defining this CPP macro we get
 # the 1.8 API.
-AC_DEFINE([H5_NO_DEPRECATED_SYMBOLS], [1], [Make sure we get the 1.8 HDF5 API])
+AC_DEFINE([H5_USE_18_API], [1], [Make sure we get the 1.8 HDF5 API])
 
 X_AC_LZ4
 X_AC_HWLOC
